docs/book/src/cronjob-tutorial/basic-project.md

```diff
@@ -28,24 +28,24 @@ First up, basic infrastructure for building you project:
 
 ## Launch Configuration
 
-We also get launch configuration under the
-[`config/`](https://sigs.k8s.io/kubebuilder.io/docs/book/cronjob-tutorial/testdata/project/config)
+We also get launch configurations under the
+[`config/`](testdata/project/config)
 directory.  Right now, it just contains
 [Kustomize](https://sigs.k8s.io/kustomize) YAML definitions required to
 launch our controller on a cluster, but once we get started writing our
 controller, it'll also hold our CustomResourceDefinitions, RBAC
 configuration, and WebhookConfigurations.
 
-[`config/default`](../TODO.md) contains a [Kustomize base](../TODO.md) for launching
+[`config/default`](testdata/project/config/default) contains a [Kustomize base](../TODO.md) for launching
```

Comment:
Same here.

Reply:
Good find @mengqiy! Unfortunately the links were valid under github review but break when using the `mdbook` generation. I looked at the `mdbook` documentation and I did not see a way to easily show directory trees of the yaml files since it is specific to generating html based on `.md`. Do you have any ideas on how to link a directory that needs to be a working link in the documentation?

Reply:
We can probably directly use `https://github.com/kubernetes-sigs/kubebuilder/tree/master/docs/book/src/cronjob-tutorial/testdata/project/config/default`. It's definitely better than a broken link :)

Reply:
I created a PR to fix the broken links, sending users to the github directory